src/less/index.less

```diff
@@ -0,0 +1,17 @@
+// Clever Styles
+// Version 0.6.2
+
+// Color and font variables
+@import "./colors.less";
+@import "./fonts.less";
+@import "./type-size.less";
+@import "./grid.less";
+
+// Component Modules
+@import "../Button/Button";
+@import "../ConfirmationButton/ConfirmationButton";
+@import "../SegmentedControl/SegmentedControl";
+@import "../TextInput/TextInput";
+@import "../CopyableInput/CopyableInput";
+@import "../Modal/Modal";
```

Comment:
We have to keep this up to date now, right? maybe the new component script should also append to this file?